client/src/transaction/client.rs

```diff
@@ -216,15 +199,16 @@ impl TxnClient {
     }
 
     // Retrieve transaction at specified block round and index.
-    pub fn get_txn(&self, round: u64, index: u32) -> BoxFuture<Option<TransactionSnapshot>> {
-        let (span, options) = self.prepare_options("TxnClient::get_txn");
-        let mut request = api::client::GetTxnRequest::new();
-        request.set_runtime_id(self.runtime_id.as_ref().to_vec());
-        request.set_round(round);
-        request.set_index(index);
+    pub fn get_tx(&self, round: u64, index: u32) -> BoxFuture<Option<TransactionSnapshot>> {
```

Comment:
Ugh.. are we using `tx` or `txn` to denote the transaction. I was using `TxnScheduler` term in go. I prefer using `txn` to `tx`, because `tx`/`rx` is ofter used to denote transmit/receive. In any case I suggest we make this consistent. Same goes for other calls (e.g. `get_tx_by_block_hash`, `get_txs`).

Reply:
I was using `tx` in go :scream: